### ISSUE TITLE
Test the new Goja async/await API

### ISSUE THREAD

**Issue by andrewslotin** [grafana/xk6-browser#685]
See: #697

**Comment by inancgumus** [comment]
It works 🎊 

```bash
# while in the xk6-browser folder
# k6-with-async is the k6 gojaAsyncAwait branch

$ XK6_K6_REPO="$PWD/../k6-with-async" xk6 build --with github.com/grafana/xk6-browser=.

$ ./k6 run script.js
# success!
```

The script:

```javascript
import { check } from 'k6';
import { chromium } from 'k6/x/browser';

export const options = {
  thresholds: {
    checks: ["rate==1.0"]
  }
}

export default async function() {
  const browser = chromium.launch();
  const context = browser.newContext();
  const page = context.newPage();

  const res = await page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
  await Promise.all([
    page.waitForNavigation(),
    page.locator('a[href="/my_messages.php"]').click(),
  ]);

  // Enter login credentials and login
  page.locator('input[name="login"]').type('admin');
  page.locator('input[name="password"]').type('123');

  await Promise.all([
    page.waitForNavigation(),
    page.locator('input[type="submit"]').click(),
  ]);
  check(page, {
    'header': page.locator('h2').textContent() == 'Welcome, admin!',
  });

  page.close();
  browser.close();
}
```

**Comment by inancgumus** [comment]
Solved by #703.
